README.md

```diff
@@ -1,202 +1,122 @@
-# Feed module - Everyone deserves RSS, Atom and Json
+# @nuxtjs/feed-next
 
-[![npm version][npm-version-src]][npm-version-href]
-[![npm downloads][npm-downloads-src]][npm-downloads-href]
-[![Github Actions CI][github-actions-ci-src]][github-actions-ci-href] 
-[![Codecov][codecov-src]][codecov-href]
-[![License][license-src]][license-href]
+## NOTE
 
-> Feed module enables everyone to have RSS, Atom and Json.
+Due to a certain [Nuxt 3 module-builder bug](https://github.com/nuxt/module-builder/issues/22), this module only works in development mode on Windows.
```

Comment:
I'm not sure this issue is affecting end-users

Reply:
This one's long been resolved, but the warning stayed there for whatever reason. It's gone